map/mode.c

```diff
@@ -11,13 +11,18 @@
 #include "mam/mam/mam_channel_t_set.h"
 
 retcode_t map_channel_create(mam_api_t *const api, tryte_t *const channel_id, const size_t depth) {
+  mam_api_channel_create(api, depth, channel_id);
+
+  // Since the current scheme to publish MAM message in our scenario we only needs to send one message per channel, so
+  // we can create a channel every time we need to send a MAM message
+  /*
```

Comment:
Why did you leave the following code?

Reply:
The changes seem to be unrelated to this subject.

Reply:
The commented one is the original code. I afraid we may need produce more channels at once, if so, we may need to use the commented code

Reply:
Don't leave the dead code. Instead, you can fix and/or clean up.